src/Video/Profiling/Query.hpp

```diff
@@ -0,0 +1,55 @@
+#pragma once
+
+#include <GL/glew.h>
+#include <cstdint>
+
+namespace Video {
+
+    /// Render surface containing framebuffers.
+    class Query {
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') class 'Query' does not have a copy constructor which is recommended since the class contains a pointer to allocated memory. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](http://localhost:9000/coding_rules#rule_key=cppcheck%3AnoCopyConstructor)

Reply:
Done